### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by codepuncher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+Then add this before it:
+```php
+/**
+ * Security settings
+ */
+ini_set('session.cookie_httponly', 'true');
+ini_set('session.cookie_secure', 'true');
+ini_set('session.use_only_cookies', 'true');
+```
```
Comment (TangRufus) [inline README.md]: these are unrelated to this plugin

**Review by tangrufus — CHANGES_REQUESTED** [review]

**Review by tangrufus — APPROVED** [review]
